src/worker/worker.ts

```diff
@@ -130,6 +133,21 @@ function applyUpdates({
   setStatus("idle")
 }
 
+function applyOptionsUpdate(newOptions: UpdatableArchipelagoParameters) {
+  setStatus("working")
+  const startTime = Date.now()
+
+  logger.debug(`Processing options update`)
+
+  const updates = archipelago.modifyOptions(newOptions)
+  console.log(updates)
```

Comment:
We probably want to remove this console.log, or at least replace it with a trace or something like that.

Reply:
🤦